script/download_k8s_binaries

```diff
@@ -1,9 +1,19 @@
 #!/usr/bin/env bash
 
+set -e
+
 kubernetes_version=$1
-staging_dir=staging/kubernetes-${kubernetes_version}
 
 main() {
+  if [ $# -ne 1 ]; then
+    echo "Usage: $(basename "$0") [KUBERNETES VERSION]"
+    exit  1
+  fi
+
+  staging_dir=$(mktemp -d)
```

Comment:
Fixed in [this commit](https://github.com/pivotal-cf-experimental/kubo-release/pull/32/commits/54315e517ab5f728af89bcb90829637b12e04c5c).